app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+    mv, file_1, path_file_2 = split_command
+
+    if "/" in command:
+        path = path_file_2[0:path_file_2.rfind("/")]
+
+        if not os.path.exists(path):
```

Comment:
Code Efficiency: You check if the path exists and create it if it doesn't. But if you try to create a directory that already exists, it will lead to an error. It's better to handle this situation properly.